ros_ws/src/autonomous/src/emergency_stop.cpp

```diff
@@ -9,23 +9,34 @@ EmergencyStop::EmergencyStop()
 
 float EmergencyStop::rangeAtDegree(const sensor_msgs::LaserScan::ConstPtr& lidar, float theta)
```

Comment:
The `rangeAtDegree` function exists twice in our codebase. I made some changes to it, but unfortunately, I only changed the one in `wall_following.cpp`. I think it would be best to reuse that one, but I'm not sure where we should put it.

Reply:
I'm neither sure where to put such a RangeAtDegree function. For now I made the issue #194 and removed the commits changing anything in the `rangeAtDegree` function.